lms/templates/dashboard.html

```diff
@@ -206,7 +206,13 @@
               %>
               <%include file='dashboard/_dashboard_course_listing.html' args='course_overview=course_overview, course_card_index=dashboard_index, enrollment=enrollment, is_unfulfilled_entitlement=is_unfulfilled_entitlement, is_fulfilled_entitlement=is_fulfilled_entitlement, entitlement=entitlement, entitlement_session=entitlement_session, entitlement_available_sessions=entitlement_available_sessions, entitlement_expiration_date=entitlement_expiration_date, entitlement_expired_at=entitlement_expired_at, show_courseware_link=show_courseware_link, cert_status=cert_status, can_refund_entitlement=can_refund_entitlement, can_unenroll=can_unenroll, credit_status=credit_status, show_email_settings=show_email_settings, course_mode_info=course_mode_info, is_paid_course=is_paid_course, is_course_blocked=is_course_blocked, verification_status=course_verification_status, course_requirements=course_requirements, dashboard_index=dashboard_index, share_settings=share_settings, user=user, related_programs=related_programs, display_course_modes_on_dashboard=display_course_modes_on_dashboard, show_consent_link=show_consent_link, enterprise_customer_name=enterprise_customer_name, resume_button_url=resume_button_url' />
             % endfor
-
+                % if not load_all_courses and courses_count < total_enrollments:
+                    <br/>
+                     ${courses_count} ${_("results successfully populated,")}
```

Comment:
@marcotuts this doesn't seem like a good message to display to the user.  Can you advise on if this is okay or should be changed?

Reply:
My suggestions / edits below:

Viewing your most recent {courses_count} enrolled courses. 
View all Enrolled Courses <--- make this look like a button, full width across the bottom of the list. 

We follow this pattern (sentence statement with count of objects, full width button with "view all" CTA)  in the discussion forums for loading all responses /comments on a thread.